flask_tutorial-code_review/main.py

```diff
@@ -0,0 +1,71 @@
+from flask import Flask, render_template, request, redirect, url_for, session, flash
+from flask_pymongo import PyMongo
+from datetime import datetime
+from flask_migrate import Migrate
+from config import DevelopmentConfig
+
+app = Flask(__name__, static_url_path='/static')
+app.config['MONGO_URI'] = 'mongodb://localhost:27017/yourdatabase'
+mongo = PyMongo(app)
+
+class Entry:
+    def __init__(self, user_id):
+        self.timestamp = datetime.utcnow()
+        self.user_id = user_id
+
+class User:
+    def __init__(self, username, password):
+        self.username = username
+        self.password = password
+@app.route('/')
+def login():
+    return render_template('login.html')
+@app.route('/index/', methods=['POST'])
+def index():
+    if request.method == 'POST':
+        username = request.form['nm']
```

Comment:
'nm' is a very bad name you need names that tells the reader their purpose

Reply:
done